bindings/python/justfile

```diff
@@ -0,0 +1,182 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Default recipe to 'help' to display this help screen
+default: help
+
+# ==============================================================================
+# Configuration & Variables
+# ==============================================================================
+
+set ignore-comments := true
+
+# Virtual environment.
+
+venv := '.venv'
+venv_bin := if os() == "windows" { venv + "/Scripts" } else { venv + "/bin" }
+
+# Tool executables, resolved within the venv.
+
+python := venv_bin + '/python'
+uv := venv_bin + '/uv'
+ruff := venv_bin + '/ruff'
+maturin := venv_bin + '/maturin'
+dprint := venv_bin + '/dprint'
```

Comment:
Why we need this? can we juse use `uv run`?

Reply:
fixed!